content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+Podczas gdy twoja aplikacja się rozwija, możesz zacząć myśleć o bardziej zintegrowanym ustawieniu. Jest [kilka JavaScriptowych narzędzi](/docs/create-a-new-react-app.html), które polecamy do rozwoju większych aplikacji. Każda z nich może wymagać drobnej lub żadnej konfiguracji i skorzystać z pełni mocy ekosystemu React.
 
-## Learn React {#learn-react}
+## Ucz się Reacta {#learn-react}
 
-People come to React from different backgrounds and with different learning styles. Whether you prefer a more theoretical or a practical approach, we hope you'll find this section helpful.
+Ludzie zaczynają z Reactem przychodząc z różnych środowisk i doświadczeń z językami programowania. Niezależnie czy preferujesz bardziej techniczne, czy praktyczne podejście, mamy nadzieję, że uznasz tę sekcję za pomocną.
 
-* If you prefer to **learn by doing**, start with our [practical tutorial](/tutorial/tutorial.html).
-* If you prefer to **learn concepts step by step**, start with our [guide to main concepts](/docs/hello-world.html).
+* Jeśli preferujesz **naukę przez praktykę**, zacznij z naszymi [praktycznymi tutorialami](/tutorial/tutorial.html).
+* Jeśli wolisz **naukę konceptów krok po kroku**, zacznij z [przewodnikiem po głównych konceptach](/docs/hello-world.html).
 
-Like any unfamiliar technology, React does have a learning curve. With practice and some patience, you *will* get the hang of it.
+Jak każda technologia, z którą nie jest się zaznajomionym, React ma swoją krzywą uczenia. Z odrobiną praktyki i cierpliwości, *zrozumiesz* na czym polega.
 
-### First Examples {#first-examples}
+### Podstawowe przykłady {#first-examples}
 
-The [React homepage](/) contains a few small React examples with a live editor. Even if you don't know anything about React yet, try changing their code and see how it affects the result.
+[Strona główna](/) Reacta zawiera kilka małych przykładów z edytorem kodu live. Nawet jeśli nie wiesz jeszcze nic o React, spróbuj zmienić kod i zobaczyć jakie to przyniesie efekty.
 
-### React for Beginners {#react-for-beginners}
+### React dla początkujących {#react-for-beginners}
 
-If you feel that the React documentation goes at a faster pace than you're comfortable with, check out [this overview of React by Tania Rascia](https://www.taniarascia.com/getting-started-with-react/). It introduces the most important React concepts in a detailed, beginner-friendly way. Once you're done, give the documentation another try!
+Jeśli wydaje Ci się, że dokumentacja Reacta prowadzi Cię w zbyt szybkim tempie, sprawdz [ten przegląd Reacta napisany przez Tanię Rascię](https://www.taniarascia.com/getting-started-with-react/). Wprawdzi Cię to w najistotniejsze koncepty Reacta w szczegółach, zachowując styl zrozumiały dla początkujących. Gdy będziesz gotowy, spróbuj ponownie przeczytać naszą dokumentację!
```

Comment:
```suggestion
Jeśli wydaje Ci się, że dokumentacja Reacta prowadzi Cię w zbyt szybkim tempie, sprawdz [ten przegląd Reacta napisany przez Tanię Rascia](https://www.taniarascia.com/getting-started-with-react/). Wprowadzi Cię on szczegółowo w najistotniejsze zagadnienia Reacta, zachowując styl zrozumiały dla początkujących. Gdy już skończysz, spróbuj ponownie przeczytać naszą dokumentację!
```

Reply:
@jakubdrozdek  Wydaje mi się, że w naszym języku odmieniamy zarówno imiona jak i nazwiska?

https://sjp.pwn.pl/poradnia/haslo/odmiana-imion-i-nazwisk-obcych;8601.html

Reply:
Masz rację, chyba z rozpędu to poprawiłem :-) Jestem za "Tanię Rascię" (btw. wyguglałem, że "Rascia" to ponoć "Raszka" po serbsku).